### DISCUSSION TITLE
remove horizontal and vertical borders of empty rows

### DISCUSSION THREAD

**Discussion by davidsaenz2003** [glideapps/glide-data-grid#575] [General]
Is it possible to remove the horizontal and vertical borders of the empty rows, and change the background color of these empty rows?
![Captura desde 2022-12-15 10-43-55](https://user-images.githubusercontent.com/114322714/207905206-b9af2888-fd94-4632-bc24-bb9f1966aa61.png)

**Comment by highviolet** [comment]
Did you find a solution for this?
